Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM alpine as make_dist
+
+RUN apk update && apk add \
+    make \
+    bash \
+    tar \
+    python3
```

Comment:
Is this `apk` stuff supposed to be here?

Reply:
Yep. This is running off of an Alpine image, and `apk` is Alpine's package manager. We need those packages.

Reply:
Ah okay, I got confused because of `FROM ubuntu:bionic` below where the build dependencies for the Debian installer are then installed.

Reply:
NP! And in case you're interested in further reading: https://docs.docker.com/develop/develop-images/multistage-build/